clang/test/CIR/CodeGen/abstract-cond.c

```diff
@@ -1,7 +1,5 @@
 // RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-cir %s -o %t.cir
 // RUN: FileCheck --check-prefix=CIR --input-file=%t.cir %s
-// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-llvm -fno-clangir-call-conv-lowering %s -o %t.ll
-// RUN: FileCheck --check-prefix=LLVM --input-file=%t.ll %s
```

Comment:
We don't want to loose the LLVM testing here, even though it might be wrong in the ABI sense for AArch64, it's still testing some important parts. Can you elaborate on why are you remove this? Should this be moved elsewhere instead?

Reply:
Oh, this is a surprise to me too. My bad. I should check the PR. BTW, I like to close this one and create a new one to make the stacked PR for https://github.com/llvm/clangir/pull/1087